src/components/veNft/VeNftContent.tsx

```diff
@@ -16,14 +17,23 @@ const VeNftContent = () => {
   const tokenSymbolDisplayText = tokenSymbolText({ tokenSymbol })
   const projectName = projectMetadata?.name ?? t`Unknown Project`
 
+  const lockDurationOptions = [
+    BigNumber.from(1),
+    BigNumber.from(7),
+    BigNumber.from(30),
+  ]
```

Comment:
**question**: Is this always gonna be static?

Reply:
No, this is read from contract starting in PR3